ansible/roles/test/files/acstests/everflow_policer_test.py

```diff
@@ -288,6 +298,6 @@ def runTest(self):
         testutils.add_filter(self.greFilter)
 
         # Send traffic and verify the mirroed traffic is rate limited
-        count = self.checkMirroredFlow()
-        assert_str = "min({1}) <= count({0}) <= max({2})".format(count, self.min_range, self.max_range)
-        assert count >= self.min_range and count <= self.max_range, assert_str
+        count, tx_pps, rx_pps = self.checkMirroredFlow()
+        assert_str = "min({1}) <= count({0}) <= max({2})".format(count, self.min_rx_pps, self.max_rx_pps)
+        assert count >= self.min_rx_pps and rx_pps <= self.max_rx_pps, assert_str
```

Comment:
@stepanblyschak why do we need pps? Why not simply use CIR?

Reply:
CIR is used to calculate accepted RX PPS range with some tolerance.
Actual RX PPS is checked to be in this range.
If youre saying that we need to calculate general rx_rate based on meter type passed then this was not supported by the test.

Reply:
> 
> 
> If youre saying that we need to calculate general rx_rate based on meter type passed then this was not supported by the test.
Yes. Exactly. But if it's not but design then no issues.